views/surveys/setup.py

```diff
@@ -0,0 +1,8 @@
+import pip
```

Comment:
You can use `pip install -r` to do the heavy lifting. https://devcenter.heroku.com/articles/python-pip shows how the syntax of a `requirements.txt` so that you can do `pip install -r requirements.txt`